subprojects/fribidi.wrap

```diff
@@ -2,6 +2,7 @@
 directory = fribidi
 url = https://github.com/fribidi/fribidi.git
 revision = master
+patch_directory = fribidi
```

Comment:
I'd prefer to avoid this if possible. Let's see if I can get a fribidi maintainer to merge some things.